tracer/src/Datadog.Trace/Agent/StatsAggregator.cs

```diff
@@ -105,6 +113,30 @@ public void AddRange(Span[] spans, int offset, int count)
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public ArraySegment<Span> ProcessTrace(ArraySegment<Span> trace)
+        {
+            if (CanComputeStats == true || CanComputeStats is null)
+            {
+                foreach (var processor in _traceProcessors)
+                {
+                    if (processor is not null)
```

Comment:
Impossible right? You instantiate them at the beginning.
That said, we could imagine initializing them lazily when we set `CanComputeStats` to true

Reply:
Yes this is impossible, it was mostly a copy/paste thing 😬 I'll remove this

Reply:
Fixed in 1b9a1dfe27e27d17d354de4bb33b6ec07b655cc8